ajax/libs/cssobj/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "cssobj",
+    "description": "Runtime CSS manager, Turn CSS into dynamic JS module, Stylesheet CRUD (Create, Read, Update, Delete) in CSSOM, Solve common problems of CSS-in-JS, name space (local) class names, media query hook for IE8, styled components",
+    "filename": "cssobj.min.js",
+    "keywords": [
+        "cssom",
+        "prefixer",
+        "css generate",
+        "diff css",
+        "media query",
+        "css modules"
+    ],
+    "author": "James Yang <jamesyang999@gmail.com>",
+    "homepage": "https://cssobj.github.io/cssobj-demo/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/cssobj/cssobj.git"
+    },
+    "npmName": "cssobj",
+    "npmFileMap": [{
+        "basePath": "dist",
+        "files": [
+            "**/cssobj.!(amd|cjs|es)*"
+        ]
+    }],
```

Comment:
Let's let the `npmFileMap` format as the followings to fit the coding style:
```js
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/cssobj.!(amd|cjs|es)*"        
      ]
    }
  ],
```
Thank you.